README.md

```diff
@@ -8,10 +8,34 @@
 yarn add --dev @wasm-tool/wasm-pack-plugin
 ```
 
+### Rust lang
+
+Rust >= `rustc 1.31.0-nightly` is required to fetch and compile rust wasm. Rust is installed and managed using [`rustup`](https://rustup.rs/)
+
 ### `wasm-pack`
 
 We expect `wasm-pack` to be in your `$PATH`. See [installation here](https://github.com/rustwasm/wasm-pack/blob/master/docs/src/setup.md#installing-wasm-pack).
 
+### `wasm-bindgen-cli`
```

Comment:
@fitzgen mentioned that this isn't needed. Could you please take a look @fitzgen

Reply:
Using the >= 0.5.0 versions of wasm-pack will automatically install the wasm-bindgen CLI for you (and the right version!) if necessary, so it shouldn't be a prerequisite to already have it installed anymore.

Just sent https://github.com/wasm-tool/wasm-pack-plugin/pull/36 too since we generally don't want to encourage `-m no-install`.

Reply:
That was because `no-install` mode was used, which forces wasm-pack into not downloading the w32-u-u target

Reply:
There have been commits since then that removed the use of no-install, btw.